occurrences/models.py

```diff
@@ -964,7 +1004,7 @@ def approve(self, send_notification=True, custom_message: Optional[str] = None):
         Raises:
             EnrolmentNotEnoughCapacityError: Not enough space for the group
         """
-        if self.occurrence.seats_taken > self.occurrence.amount_of_seats:
+        if not self.occurrence.has_space_left():
```

Comment:
`a > b` here has been changed to `not(a < b)` i.e. `a >= b` so the implementation has changed. Is this intentional?

Reply:
Good catch, because I was first going to say that it was intentional, but then again, I started to think where it was used and how that function really should check it and I came to conclusion, that this line of code really needs to test whether the change that was already made still fits in the defined limits. So yes, this would have changed the feature so that 1 empty place would have always be needed. I'll write a new test for this too.

Reply:
I reverted this line's code change and improved an old test related to this so that it tests this case too.